build-settings-logic/settings.gradle.kts

```diff
@@ -2,6 +2,14 @@
  * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
  */
 
+pluginManagement {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
```

Comment:
I think this `pluginManagement` block can be removed as we don't apply any plugins from custom repositories to the `build-settings-logic` project